src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/HealthCheckSpecification.groovy

```diff
@@ -145,18 +149,26 @@ class HealthCheckSpecification extends HealthCheckBaseSpecification {
                 }
             }
         }
-        withPool {
-            [healthCheckEndpoint,
-             allZookeeperNodesInExpectedState,
-             allSwitchesAreActive,
-             allLinksAreActive,
-             noFlowsLeft,
-             noLinkPropertiesLeft,
-             linksBandwidthAndSpeedMatch,
-             noExcessRulesMeters,
-             allSwitchesConnectedToExpectedRegion,
-             featureTogglesInExpectedState,
-             switchesConfigurationIsCorrect].eachParallel { it() }
+
+        if(enable) {
+            withPool {
+                [healthCheckEndpoint,
+                 allZookeeperNodesInExpectedState,
+                 allSwitchesAreActive,
+                 allLinksAreActive,
+                 noFlowsLeft,
+                 noLinkPropertiesLeft,
+                 linksBandwidthAndSpeedMatch,
+                 noExcessRulesMeters,
+                 allSwitchesConnectedToExpectedRegion,
+                 featureTogglesInExpectedState,
+                 switchesConfigurationIsCorrect].eachParallel { it() }
+            }
+        } else {
+            withPool {
+                [healthCheckEndpoint,
+                 allZookeeperNodesInExpectedState].eachParallel { it() }
```

Comment:
Decided to keep these two basic checks to avoid unnecessary failure investigation.